project/src/main/java/com/google/sps/utilities/WrapperDatabaseService.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.utilities;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+
+/** Class that initializes spanner and database client. */
+public class WrapperDatabaseService implements DatabaseService {
+  private final Spanner spanner;
+  private final DatabaseClient databaseClient;
+
+  public WrapperDatabaseService() {
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    spanner = options.getService();
+    DatabaseId database =
+        DatabaseId.of(options.getProjectId(), DatabaseConstants.INSTANCE_ID, DatabaseConstants.DATABASE_ID);
+    databaseClient = spanner.getDatabaseClient(database);
```

Comment:
Could you move these to an initialize function?

Reply:
Ack, no longer applies - may be useful in future PRs.